DESCRIPTION

```diff
@@ -57,7 +57,7 @@ RdMacros:
     lifecycle
 Remotes: 
     r-lib/gert,
-    r-lib/gh
+    r-lib/gh#139
```

Comment:
This is going to fail as soon as the PR is closed. It's a bit safer (but not as descriptive) to refer to a branch with the `@branch` notation instead.

Reply:
I don't plan for this situation to last very long, but yes you're right.

Reply:
Both are going to break when we merge and delete the branch. I'll update this as soon as we merge in gh.